### PR TITLE
Remove EOL Python Versions

### PR DESCRIPTION
**Pull request by jamescurtin**
According to the [Python release schedule](https://devguide.python.org/#status-of-python-branches), Python 3.4 is past end of life, and Python 3.7 has entered into bugfix status. This PR updates the package to reflect this.

### CODE REVIEW

**Comment by insspb** [comment]
Everything looks correct. I leave this comment just to see when it will be answered by the core team or merged. Unfortunately, core development looks dead.

**Comment by jamescurtin** [comment]
HI, @ssbarnea, I do not have write access to this repo. Please merge if you can. Thanks!

**Comment by ssbarnea** [comment]
@pydanny safe to merge.

File: docs/contributor_guidelines.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ Before you submit a pull request, check that it meets these guidelines:
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a docstring, and add the
    feature to the list in README.rst.
-3. The pull request should work for Python 2.7, 3.3, 3.4, 3.5, 3.6, and PyPy on
+3. The pull request should work for Python 2.7, 3.3, 3.5, 3.6, 3.7, and PyPy on
```
Comment (insspb) [inline docs/contributor_guidelines.rst]: python 3.3 should be removed too

**Comment by insspb** [comment]
https://img.shields.io/pypi/pyversions/cookiecutter.svg change should be checked too after this merged.

**Comment by insspb** [comment]
Closed as a duplicate to speed up the review process. Everything from here merged to #1181

**Pull request closed without merging**